zoo/settings.py

```diff
@@ -78,6 +78,7 @@
     'default': {
         'ENGINE': 'django.db.backends.postgresql_psycopg2',
         'NAME': 'supreme',
+        'USER': 'sandeep',
```

Comment:
Sorry, I have not specified a default user name in the settings.py. Currently I work by setting environment variables. I am not sure if that's a preferred way or not. I will try to make things easy for you here.